lib/VM/JSLib/print.cpp

```diff
@@ -33,6 +35,14 @@ CallResult<HermesValue> print(void *, Runtime *runtime, NativeArgs args) {
     llvh::outs() << StringPrimitive::createStringView(
                         runtime, runtime->makeHandle(std::move(*res)))
                         .getUTF16Ref(tmp);
+
```

Comment:
This change was added for debugging ... And it's not intended for final checkin ..